lib/Throttle.php

```diff
@@ -88,6 +95,7 @@ public function applyBruteForcePolicyForLogin($uid, $ip) {
 		$thresholdTime = $lastAttempt - $this->config->getBruteForceProtectionTimeThreshold();
 		if ($this->loginAttemptMapper->getFailedLoginCountForUidIpCombination($uid, $ip, $thresholdTime) >=
 			$this->config->getBruteForceProtectionFailTolerance()) {
+			$this->logger->warning('User \''. $uid .'\' is blocked for '. $this->parseBanPeriodForHumans($banPeriod) .'. Too many failed login attempts', ['app' => 'brute_force_protection']);
```

Comment:
```suggestion
			$this->logger->warning('User \''. $uid .'\' is blocked for'. $this->parseBanPeriodForHumans($banPeriod) .'. Too many failed login attempts', ['app' => 'brute_force_protection']);
```

`parseBanPeriodForHumans` seems to put a leading space in the returned string, so we do not need a space after "for".